CONTRIBUTING.rst

```diff
@@ -254,6 +255,14 @@ Develop the fix for the job logic and test it
    you are fixing There may be other experimental jobs in the queue that are
    failing, but you can ignore those.
 
+.. note::
+
+  If a patch set is already in the cisco-experimental queue leaving another
+  ``cisco-experimental`` comment will **NOT** cause that job to restart. If you
+  notice an intermittent error and want to restart the experimental queue run
+  the best option is to rebase and submit another patch set to your [DNM]
+  patch.
```

Comment:
to your [DNM] patch, as new patch-sets will clear all previous jobs from the queue.